content/src/content/jcr_root/apps/core/wcm/components/image/v2/image/clientlibs/editor/js/image.js

```diff
@@ -118,7 +118,9 @@
                 $altGroup.show();
                 $linkURLGroup.show();
             }
-            $linkURLField.adaptTo("foundation-field").setDisabled(checkbox.checked);
+            if ($linkURLField.length) {
```

Comment:
how about checking if the field exists?
`if ($linkURLField) {`

Reply:
It's a jQuery object so it will always exist, except the length will be 0.